jvm/src/test/scala/cats/tests/FutureTests.scala

```diff
@@ -0,0 +1,39 @@
+package cats
+package jvm
+package tests
+
+import cats.data.Xor
+import cats.laws.discipline._
+import cats.tests.CatsSuite
+
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import org.scalacheck.Arbitrary
+import org.scalacheck.Arbitrary.arbitrary
+
+class FutureTests extends CatsSuite {
+  val timeout = 3.seconds
+
+  def futureXor[A](f: Future[A]): Future[Xor[Throwable, A]] =
+    f.map(Xor.right[Throwable, A]).recover { case t => Xor.left(t) }
+
+  implicit def eqfa[A: Eq]: Eq[Future[A]] =
```

Comment:
should we add this but have it take a timeout to the object where we define the `Monad`?

`def futureEq[A: Eq](timeout: Duration): Eq[Future[A]] = ...`